dev-util/kdstatemachineeditor/kdstatemachineeditor-1.2.1.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils kde5-functions multilib
+
+QT_MINIMAL="5.3"
```

Comment:
This is unnecessary. Qt-5.3 is ancient, so just leave it at the eclass' default.